src/runtime_src/core/common/system.cpp

```diff
@@ -63,4 +63,16 @@ get_mgmtpf_device(device::id_type id)
   return instance().get_mgmtpf_device(id);
 }
 
+std::pair<uint64_t, uint64_t>
+get_total_devices()
+{
+  return instance().get_total_devices();
+}
+
+void
```

Comment:
Can you return the index?

Reply:
I thought we wanted all void functions

Reply:
haha :-)  Sorry for the confusion.  No, what we don;'t want is any functions to return an error code, but they can surely return a result and throw on error.

Reply:
Oh. Ok that makes sense!